src/main/java/mate/jdbc/dao/impl/CarDaoImpl.java

```diff
@@ -0,0 +1,218 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.lib.exception.DataProcessingException;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+
+    @Override
+    public Car create(Car car) {
+        String query = "INSERT INTO cars (name, manufacturer_id) "
+                + "VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createCarStatement =
+                        connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
+            createCarStatement.setString(1, car.getName());
+            createCarStatement.setObject(2, car.getManufacturer());
+            createCarStatement.executeUpdate();
+            ResultSet resultSet = createCarStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                car.setId(resultSet.getObject(1, Long.class));
+            }
+            return car;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't create car " + car, e);
+        }
+    }
+
+    @Override
+    public Car get(Long id) {
+        String query = "SELECT c.id AS car_id, "
+                + " c.name AS car_name, "
+                + " m.id AS manufacturer_id, m.country, "
+                + " m.name AS manufacturer_name, m.country "
+                + " FROM taxi_my.cars c JOIN taxi_my.manufacturers m ON c.manufacturer_id = m.id "
```

Comment:
```suggestion
                + " FROM cars c JOIN taxi_my.manufacturers m ON c.manufacturer_id = m.id "
```
у всіх запитах потрібно видалити `taxi_my`